src/proxy/src/main/java/ee/ria/xroad/proxy/monotoring/MonitorAgent.java

```diff
@@ -42,22 +46,24 @@ public final class MonitorAgent {
     private MonitorAgent() {
     }
 
-    /**
-     * Initialize the MonitorAgent with given ActorSystem.
-     * This method must be called before any other methods in this class.
-     * @param actorSystem actor system to be used by this monitoring agent
-     */
-    public static void init(ActorSystem actorSystem) {
-        monitorAgentImpl = new DefaultMonitorAgentImpl(actorSystem);
+    public static void init() throws Exception {
+        init(getGrpcInternalHost(), getProxyGrpcPort(), getSignerClientTimeout());
```

Comment:
MonitorAgent should have its own timeout property, not the getSignerClientTimeout().